csrc/tensor_view.cpp

```diff
@@ -1370,6 +1370,36 @@ void TensorView::applyMmaSwizzleForTMALoad(
       this, swizzle, permute_outer_dim);
 }
 
+void TensorView::scheduleMmaTMAStore() {
```

Comment:
Should we not make this a member function of `TensorView`? Unlike swizzleTMABox, which is generally useful for all TMA operations of arbitrary box size, this is too specificly tied to the 2D tensor of matmul.

Reply:
That makes sense - I was conflicted about putting it here. Would it make sense to put it in mma_utils?

Reply:
> Would it make sense to put it in mma_utils?

That sounds good